templates/extensions/the-graph/packages/subgraph/networks.json

```diff
@@ -0,0 +1,7 @@
+{
+  "localhost": {
+    "YourContract": {
+      "address": "0x5FbDB2315678afecb367f032d93F642f64180aa3"
```

Comment:
should it be here?

Reply:
https://github.com/scaffold-eth/create-eth/pull/25#discussion_r1590007269